TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
I think it is a big issue!
Much like the `TYPO3.Party` package I try to avoid the `NodeTypes` package where possible and I came across a couple of agencies and Neos devs that do the same (why do you think it's just 1%?)
It's a good place to get started but at least for a complex scenario it will never exactly fit and rewiring/disabling existing NodeTypes can get quite cumbersome and confusing